lib/line_collection.rb

```diff
@@ -66,14 +65,12 @@ def index_at(x)
   end
 
   class LineCollection
-    attr_reader :text_length, :lines
+    attr_reader :lines
 
     include Enumerable
 
-    def initialize(font, size_enum, lines = [])
+    def initialize(lines = [])
       @lines = lines
-      @font = font
-      @size_enum = size_enum
```

Comment:
Actually `LineCollection` didn't need any size calculation anymore so I just removed the references to font and size